elastic/kibana
# [SecuritySolution] Dashboard listing UI

## Summary


1. Align dashboard listing UI with Kibana dashboard.
2. `Security Solution` tags are selected by default and removable by users.

**Prerequisite:**
This PR is waiting for https://github.com/elastic/kibana/pull/160871 to be merged


**Steps to verify:**
1. Visit Security > Dashboards, and create a dashboard from this page.
2. Back to Security Dashboards page, you should see the dashboard you just created and Security Solution tag should be selected by default in the tag filters.
3. Open the tag options, click the Security Solution tag. Observe that it should be removable, and it should display all the dashboards you have in the table.

**Known issues:**
https://github.com/elastic/kibana/pull/160540#issuecomment-1610395834

**Before:**

<img width="2545" alt="Screenshot 2023-06-27 at 09 24 19" src="https://github.com/elastic/kibana/assets/6295984/bc0fa0b1-96ad-43b0-afc1-48444dfb5691">


**After:**
<img width="2543" alt="Screenshot 2023-06-27 at 09 22 21" src="https://github.com/elastic/kibana/assets/6295984/82d0a868-bda2-431f-b0b5-9cbc34d3ae71">


### Checklist

Delete any items that are not applicable to this PR.
